scripts/jenkins/wait-for-cluster.sh

```diff
@@ -0,0 +1,4 @@
+until kubectl cluster-info
```

Comment:
Always good to documented anyways what the result of the offline discussion was for everyone else to understand and for us to understand when we go back.

For the reference: there was only the `until` statement and the `sleep` function so there was nothing to error.

However, as this script is now more than that, I propose that `set -e` is added. Plus, even if we don't see a possible error condition doesn't mean there isn't one :wink: